src/http.js

```diff
@@ -25,6 +25,7 @@ export default function http(url, request) {
   if (request.requestInterceptor) {
     request = request.requestInterceptor(request) || request
   }
+  request.credentials = 'same-origin';
```

Comment:
@larshp can you move this line before the requestInterceptor runs? Also, would be great if you add a quick test.